### ISSUE TITLE
Support SRV lookups

### ISSUE THREAD

**Issue by jhrozek** [jhrozek/pam_hbac#34]
Would be required for larger environment anyway

**Comment by code-with-amitk** [comment]
Hello,
Can i take up this Issue?
Thanks

**Comment by jhrozek** [comment]
Yes, but because SRV lookups typically yield multiple records, it would be nice to implement issue #33 first and support some kind of failover between multiple records. Or did you plan to support just the first server returned from SRV query?
